textworld/generator/text_generation.py

```diff
@@ -414,19 +414,26 @@ def assign_description_to_quest(quest, game, grammar):
             actions_desc, _ = generate_instruction(quest.actions[-1], grammar, game.infos, game.world, counts)
             only_one_action = True
         else:
-            actions_desc = ""
+            actions_desc_list = []
             # Decide if we blend instructions together or not
             if grammar.options.blend_instructions:
                 instructions = get_action_chains(quest.actions, grammar, game.infos)
             else:
                 instructions = quest.actions
 
             only_one_action = len(instructions) < 2
+            ensure_next_lowercase = False
```

Comment:
Is the `ensure_next_lower_case` still needed since the regex below changes the case of all characters following a point?

Reply:
It's to prevent things like `Then,Insert the key into the locker's lock to unlock it.` (4-th example above)

Reply:
That's actually a typo in `house_instruction.twg` (https://github.com/Microsoft/TextWorld/blob/master/textworld/generator/data/text_grammars/house_instruction.twg#L52). `Insert` on that line should have been all lowercase. Can you fix it and consider the changes I've suggested?

Reply:
Here's another example, as a result of `--blend-instructions`:
```
>>> tw-make custom --world-size 5 --nb-objects 25 --quest-length 25 --seed 1237 --blend-instructions --output gen_games/
Global seed: 1237
Game generated: gen_games/tw-game-PLsBH6H2-house-Gae-y5o3f0af12OtKQa.ulx
```
```

The intro text shows:
"First off, venture east.And then,Recover the cuboid
keycard and use it to unlock the cuboid box.And then,ensure that the cuboid box
within the office is open. That's it!"

Reply:
That's another typo in `house_instruction.twg` https://github.com/Microsoft/TextWorld/blob/master/textworld/generator/data/text_grammars/house_instruction.twg#L81. `#pick-up_synonym_v#` should have been `#pick-up_synonym_1#`. Can you edit it as well?